File: data_baseline/openrouter__qwen__qwen3-coder-plus/scala/rep1/SessionManager.scala
import scala.collection.mutable
import java.util.UUID

// Simple in-memory session storage
class SessionManager {
  private val activeSessions = mutable.Map[String, Int]()  // sessionId -> userId
  
  def createSession(userId: Int): String = {
    val sessionId = generateSessionId()
    activeSessions.put(sessionId, userId)
    sessionId
  }
  
  def getUserIdForSession(sessionId: String): Option[Int] = {
    // In a real app, we'd also verify session expiration here
    activeSessions.get(sessionId)
  }
  
  def destroySession(sessionId: String): Unit = {
    activeSessions.remove(sessionId)
  }
  
  private def generateSessionId(): String = {
    UUID.randomUUID().toString
  }
}

File: data_baseline/openrouter__qwen__qwen3-coder-plus/java/rep4/src/com/todoserver/SessionManager.java
package com.todoserver;

import java.util.Map;
import java.util.UUID;
import java.util.concurrent.ConcurrentHashMap;

public class SessionManager {
    private final Map<String, Session> sessions = new ConcurrentHashMap<>();
    
    public String createSession(int userId) {
        String sessionId = generateSessionToken();
        sessions.put(sessionId, new Session(userId));
        return sessionId;
    }
    
    public Session getSession(String sessionId) {
        if (sessionId == null) {
            return null;
        }
        return sessions.get(sessionId);
    }
    
    public boolean logoutSession(String sessionId) {
        if (sessionId == null) {
            return false;
        }
        return sessions.remove(sessionId) != null;
    }
    
    private String generateSessionToken() {
        return UUID.randomUUID().toString();
    }
    
    static class Session {
        final int userId;
        
        Session(int userId) {
            this.userId = userId;
        }
    }
}